llvm/lib/Target/AMDGPU/SIInstrInfo.cpp

```diff
@@ -2812,6 +2812,10 @@ MachineInstr *SIInstrInfo::commuteInstructionImpl(MachineInstr &MI, bool NewMI,
     swapSourceModifiers(MI, Src0, AMDGPU::OpName::src0_modifiers,
                         Src1, AMDGPU::OpName::src1_modifiers);
 
+    if (isSDWA(MI))
```

Comment:
It's not just SDWA - VOP3 instructions with OPSEL also use src0/1_sel

Reply:
will fix